.gitlab-ci.yml

```diff
@@ -15,7 +15,7 @@ build-package:
   stage: build
   image: $IMAGE_BUILD
   before_script:
-    - go install github.com/gohugoio/hugo@v0.92.0
+    - go install --tags extended github.com/gohugoio/hugo@v0.92.1
```

Comment:
I think need to set the CGO, check the docs @spirosoik

Reply:
@spirosoik - Following up on this feedback from @cpanato. Is action required for this step prior to merging this into production?

Reply:
Confirmed with @spirosoik that this has been addressed.